composer.json

```diff
@@ -123,6 +123,7 @@
     "symfony/web-profiler-bundle": "^6.2",
     "symfony/workflow": "^6.2",
     "symfony/yaml": "^6.2",
+    "symfony/webpack-encore-bundle": "^1.7",
```

Comment:
@robertSt7 that would be a `-dev` requirement right? 
And maybe we should add it to `AdminBundle/composer.json` because actually it's not a `pimcore/pimcore` requirement, right?

Reply:
We can't move it to the `-dev` requirements, because we need it to import the files in the templates. 

resolved by https://github.com/pimcore/pimcore/pull/14399